app/controllers/incomeSources/manage/ManageObligationsController.scala

```diff
@@ -130,59 +149,53 @@ class ManageObligationsController @Inject()(val checkSessionTimeout: SessionTime
               isAgent = true,
               taxYear,
               changeTo,
-              ""
+              Some("")
```

Comment:
Some("") is really a None as there is no value, we have quite a lot of these in our code base, so need to tidy up at some point